website/content/docs/ecs/terraform/secure-configuration.mdx

```diff
@@ -136,8 +136,7 @@ resource "aws_secretsmanager_secret_version" "gossip_key" {
 
 ### Enable secure deployment
 
-Add the following configurations to enable secure deployment. The `acl_secret_name_prefix`
-should be the same as the `name_prefix` you provide to the ACL controller module.
+Add the following configurations to enable secure deployment.
```

Comment:
The example configuration describes more than just enabling audit logging. This change was included because the `acl_secret_name_prefix` field was removed from the `mesh-task` module in an unrelated PR.

What are your thoughts on:

```
To enable secure deployment, add the following configuration.
```
?

Reply:
OK, updated per the suggested wording. The `acl_secret_name_prefix` is no longer supported by the mesh-task module and has been removed. So we've omitted reference to it here.